src/lib/viewers/image/ImageBaseViewer.js

```diff
@@ -311,23 +310,20 @@ class ImageBaseViewer extends BaseViewer {
     }
 
     /**
-     * Handles image element loading errors.
+     * Handles a content download error
      *
-     * @private
-     * @param {Error} err - Error to handle
+     * @param {Error} err - Load error
+     * @param {string} imgUrl - URL we are using as the image src
      * @return {void}
      */
-    errorHandler(err) {
-        /* eslint-disable no-console */
-        console.error(err);
-        /* eslint-enable no-console */
```

Comment:
Did you mean to take these out? They're stripped out on the production build but are occasionally useful on dev.

Reply:
yep, I forgot these are stripped out of the prod build, thanks.